src/components/HoverCard/HoverCardOnUsername.tsx

```diff
@@ -73,16 +75,30 @@ export default function HoverCardOnUsername({ userRef, queryRef }: Props) {
   const [isHovering, setIsHovering] = useState(false);
 
   const deactivateHoverCardTimeoutRef = useRef<ReturnType<typeof setTimeout> | null>(null);
+  const activateHoverCardTimeoutRef = useRef<ReturnType<typeof setTimeout> | null>(null);
   const handleMouseEnter = () => {
     if (deactivateHoverCardTimeoutRef.current) {
       clearTimeout(deactivateHoverCardTimeoutRef.current);
     }
 
-    setIsActive(true);
-    setIsHovering(true);
+    /**
+     * We don't want the popover to trigger instantly. Otherwise
+     * users will trigger the popup accidentally while just moving
+     * their mouse around.
+     *
+     * This ensures that the popover is opened with absolute intent.
+     */
```

Comment:
important callout here. it felt better with a slight delay when moving my mouse around the modal

Reply:
AMAZING catch 🧠

Reply:
holy shit. Definitely much better!